src/front/ApplicationFactory.ts

```diff
@@ -8,6 +8,9 @@ import { RootController } from './controllers/RootController';
 import { AuthController } from './controllers/AuthController';
 import { ErrorController } from './controllers/ErrorController';
 import { SessionFactory } from './middlewares/SessionFactory';
+import { logging } from '../common/logger/Logger';
+
+const logger = logging.getLogger();
```

Comment:
**BUG** to bedzie powodowalo, ze na potrzeby kazdego modulu bedzie tworzony osobny logger - logger powinien byc jeden i jego instancja powinna byc brana z DI